### PR TITLE
test: cache playwright browsers in Github CI/CD

### PR DESCRIPTION
**Pull request by TrickyPi**
related #138

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npx lint-staged
+# npx lint-staged
```
Comment (jpillora) [inline .husky/pre-commit:4]: Drop comments?

Comment (jpillora) [reply .husky/pre-commit]: Sounds good

**Comment by TrickyPi** [comment]
Caching browser binaries can reduce playwright browser installation time, but not very well on Linux. But on windows, it can reduce the time by about 30s.

**Comment by jpillora** [comment]
https://github.com/microsoft/playwright/issues/7249#issuecomment-1317670494

See this comment? Not sure if it helps

**Comment by jpillora** [comment]
Oh maybe that’s yarn only

that thread has some good information though

**Comment by TrickyPi** [comment]
Yeah, I've read this thread(this help me a lot).I don't think using yarn or npm will affect the playwright install time. and the code in the comment above doesn't work on github actions, becasuse OS dependencies of playwright doesn't exist if hitting playwright browser cache, and i haven't found a way to cache OS dependencies.

**Pull request merged**